lua/nvim-tree/utils.lua

```diff
@@ -328,4 +330,25 @@ function M.key_by(tbl, key)
   return keyed
 end
 
+---Execute fn timeout ms after the last invocation with context
+---@param context string identifies the callback to debounce
+---@param timeout number ms to wait
+---@param fn function callback to execute on completion
+function M.debounce(context, timeout, fn)
+  if M.debouncers[context] then
+    M.debouncers[context]:close()
```

Comment:
i'd pcall this in case this is already closing somehow.

Reply:
Done.

`pcall(uv.close, M.debouncers[context])` looks strange; `uv.close` is actually `uv_handle_t` close.

It's OK if the call fails, the timer will be released after this.